Software/Python/setup.py

```diff
@@ -12,7 +12,26 @@
     name="gopigo3",
     description="Drivers and examples for using the GoPiGo3 in Python",
     author="Dexter Industries",
-    url="http://www.dexterindustries.com/GoPiGo/",
+    url="http://www.dexterindustries.com/GoPiGo3/",
     py_modules=['gopigo3','easygopigo3','I2C_mutex'],
```

Comment:
Could we move this `py_modules` list next to the `packages` list?
It's going to be more readable because we batch all the things with a common denominator together.